### ISSUE TITLE
SLES 12 RPM Packaging

### ISSUE THREAD

**Issue by mattaezell** [globus/globus-toolkit#64]
I have a new SLES 12 system that I would like to use GT6 on.  Most of the SLES 11 packages seem to work, but gridftp-server requires the xio-udt-driver, which requires libffi43.  SLES 12 ships libffi4, which seems to be libffi48 under the hood.  Has anyone looked into what it would take to build native RPMs for SLES12?

**Comment by danpowers** [comment]
Hi Matt,

At the present time, SLES 12 is not a supported platform for GT6. However, we're currently looking in to what demand exists for such support and what it would take to add SLES 12 support for GT6. As such, we'd appreciate it if you could tell us as much about your use case as possible. We're particularly interested in anything you can tell us about why your deployment/project has chosen to use SLES 12. 

-Regards

Dan Powers

**Comment by mattaezell** [comment]
Hello Dan- I'm using Globus on a new Cray machine.  As of Cray Linux Environment 6.0, SLES12 is the only supported OS.

Thanks,
~Matt

**Comment by bester** [comment]
I've started updating our build process so that we can generate SLES 12 RPMs.

**Comment by bester** [comment]
I've completed a compile of the toolkit for SLES12.

I've updated the [RPM repo package](http://toolkit.globus.org/ftppub/gt6/installers/repo/globus-toolkit-repo-latest.noarch.rpm) to include SLES 12 repositories.

Install that, then do

    $ sudo zypper mr -e -r Globus-Toolkit-6-Unstable-sles12

to enable/refresh that repo and you will have access to the initial packages. We'll do more testing before moving those packages into the stable repository.
